hax/hax/queue/cli.py

```diff
@@ -49,7 +46,6 @@ def parse_opts(ctx, queue: str, type: str, payload: str):
 
 def main():
     _setup_logging()
-    inject.configure(di_configuration)
```

Comment:
Why do you remove this?

Reply:
Yes this was rebase related issue.
Resolved!